src/Card.js

```diff
@@ -19,19 +19,40 @@ export default class Card extends Component {
 
 
   render() {
+    console.log('starring', this.props.episode.starring)
+    if (this.props.episode.starring.includes('Willow')) {
     return (
       <li className="card">
       <div className="background" style={{ backgroundImage: `url(${this.props.episode.image})` }}></div><h3 className="episode-name">
         {
           this.props.episode.name
         }
         </h3>
+        <h1 className="card-title">Buffy the Vampire Slayer</h1>
         <div className="caption">
           <h3>Season {this.props.episode.season}</h3>
           <p>{this.props.episode.synopsis}</p>
           <i className="fas fa-plus" onClick={this.handleState}></i>
         </div>
       </li>
-    );
+    )
+  } else {
+    return (
+      <li className="card">
+      <div className="background" style={{ backgroundImage: `url(${this.props.episode.image})` }}></div><h3 className="episode-name">
+        {
+          this.props.episode.name
+        }
+        </h3>
+        <h1 className="card-title">Angel</h1>
+        <div className="caption">
+          <h3>Season {this.props.episode.season}</h3>
+          <p>{this.props.episode.synopsis}</p>
+          <i className="fas fa-plus" onClick={this.handleState}></i>
+        </div>
+      </li>
+    )
+
   }
-}
\ No newline at end of file
+}
+}
```

Comment:
I like the if/else, but there is also quite a bit being repeated. Maybe we can try and dry this up a bit later.